Update psammead dependencies

ChrisBAshton opened the pull request:
Resolves #1172

**Overall change:** Pulls in latest Psammead dependencies. Needed for latest caption styling.

**Code changes:**

- Updated all latest psammead dependencies except `gel-foundations` (current version is buggy on Psammead)
- Resolved `js-yaml` subdependencies to be a version that doesn't have a vulnerability
- Updates snapshots (Psammead components have updated to use `rem` over `em`)

Changes from change logs:

```
"@bbc/psammead-brand": Uses rem for media queries.
"@bbc/psammead-caption": Remove background colour, update colour to Cloud Dark. Use font style italic. Also ensures left/right padding remains tight in context of article.
"@bbc/psammead-copyright": Updates dependencies
"@bbc/psammead-image": n/a (Update storybook badge url)
"@bbc/psammead-image-placeholder": n/a (Update storybook badge url)
"@bbc/psammead-inline-link": n/a (Update storybook badge url)
"@bbc/psammead-paragraph": increases padding-bottom as per designs
"@bbc/psammead-sitewide-links": uses rem for media queries.
"@bbc/psammead-visually-hidden-text": n/a (Update storybook badge url)
```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

ChrisBAshton committed:
update psammead packages (incl caption styles)

Also fixes audit by running 'npm update js-yaml --depth 6'

ChrisBAshton committed:
update snapshots after updating psammead dependencies

dr3 commented on package.json:
```diff
@@ -78,20 +78,20 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.3.4",
-    "@bbc/gel-foundations": "^0.2.4",
+    "@bbc/gel-foundations": "^0.3.0",
```
Last time I did this James wanted a list of all possible changes to these packages like https://github.com/bbc/simorgh/pull/1394. Its made by comparing the versions and manually copying the changes in their changelog

dr3 reviewed (dismissed):
![](https://media.giphy.com/media/l46CwVyMaDnl1pRhC/source.gif)

Commit:
bump inline link to 0.3.4

Commit:
Merge branch 'latest' of github.com:bbc/simorgh into update-psammead-dependencies

Commit:
snapshots

sareh reviewed (dismissed):
👍 Looks good. I noted that now that we have removed the background colour of the caption, the padding (1rem, 16px) makes the text look indented. 

<img width="788" alt="Screen Shot of caption with 1rem padding" src="https://user-images.githubusercontent.com/3028997/54907707-13003e00-4ede-11e9-969a-15745de7aca0.png">

We'll likely want to check with UX whether this is desired.

ChrisBAshton commented:
Failed UX review, raised bug with gel-foundations: https://github.com/bbc/psammead/issues/388

ChrisBAshton commented:
> 👍 Looks good. I noted that now that we have removed the background colour of the caption, the padding (1rem, 16px) makes the text look indented.
> 
> <img alt="Screen Shot of caption with 1rem padding" width="788" src="https://user-images.githubusercontent.com/3028997/54907707-13003e00-4ede-11e9-969a-15745de7aca0.png">
> 
> We'll likely want to check with UX whether this is desired.

We've now raised https://github.com/bbc/psammead/issues/382 for this.

ChrisBAshton commented:
Given we've still work to do on figcaption, let's fix https://github.com/bbc/psammead/issues/382 before continuing with this PR.

ChrisBAshton commented:
Blocked on https://github.com/bbc/psammead/pull/393.

ChrisBAshton committed:
merged with latest

ChrisBAshton committed:
downgrade to @bbc/gel-foundations@0.2.5

0.3.0 has a buggy breakpoint, to be fixed: https://github.com/bbc/psammead/issues/388

ChrisBAshton committed:
updates snapshots having downgraded to gel-foundations@0.2.5

ChrisBAshton committed:
Merge branch 'latest' into update-psammead-dependencies

ChrisBAshton committed:
Merge branch 'latest' into update-psammead-dependencies

ChrisBAshton committed:
pulls in latest caption/copyright components

ChrisBAshton committed:
Merge branch 'update-psammead-dependencies' of https://github.com/BBC-News/simorgh into update-psammead-dependencies

ChrisBAshton commented:
Now blocked on #398.

ChrisBAshton commented:
Now blocked on https://github.com/bbc/psammead/pull/381.

ChrisBAshton commented:
Now blocked on https://github.com/bbc/psammead/pull/406.

ChrisBAshton committed:
latest dependencies'

ChrisBAshton committed:
merged with latest

ChrisBAshton committed:
updates snapshots now components use rem for breakpoints

pjlee11 reviewed (dismissed):
LGTM 👍

sareh committed:
Merge branch 'latest' into update-psammead-dependencies

ChrisBAshton commented:
Blocked on https://github.com/bbc/psammead/pull/410 :(

ChrisBAshton committed:
merge with latest

ChrisBAshton committed:
bumps psammead-caption

sareh commented on .env:
```diff
@@ -1,4 +1,7 @@
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```
Can you please revert this file's changes?

ChrisBAshton committed:
revert .env

sareh reviewed (approved):
👍 Looks great.

dr3 commented:
Have popped this straight into ready for test, This can be UX reviewed in parallel. Any changes will likely be raised in new issues

jamesbrumpton committed:
Merge branch 'latest' into update-psammead-dependencies

ChrisBAshton committed:
Merge branch 'latest' into update-psammead-dependencies
